### PR TITLE
Add registry mirror configurations for containerd

### PR DESCRIPTION
**Pull request by zoetrope**
Signed-off-by: zoetrope <a.ikezoe@gmail.com>

### CODE REVIEW

File: ignitions/common/systemd/k8s-containerd.service [hunk 1]
```diff
@@ -19,6 +19,7 @@ Environment="HTTPS_PROXY=http://{{ index .Spec.IPv4 0 }}:30128"
 Environment="HTTP_PROXY={{ Metadata "proxy_url" }}"
 Environment="HTTPS_PROXY={{ Metadata "proxy_url" }}"
 {{ end }}
+Environment="NO_PROXY=*.registry.{{ if eq (or .Spec.Labels.datacenter "dc1") "dc1" }}gcp0.dev-ne.co{{ else }}{{ .Spec.Labels.datacenter }}.cyozu-ne.co{{ end }}"
```
Comment (ymmt2005) [inline ignitions/common/systemd/k8s-containerd.service:22]: Let's use `10.0.0.0/8`.

**Review by ymmt2005 — CHANGES_REQUESTED** [review]

**Review by ymmt2005 — DISMISSED** [review]
LGTM

File: ignitions/common/systemd/k8s-containerd.service (hunk 1 above)
Comment (zoetrope) [reply ignitions/common/systemd/k8s-containerd.service:22]: It's not working...

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Pull request merged**